backend/src/main/java/io/github/orionhealth/xbdd/model/common/User.java

```diff
@@ -2,9 +2,11 @@
 
 import java.util.Map;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.gson.annotations.SerializedName;
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
As an fyi we should probably stick these on all our models that can have null values. That way the JSON object will just not have that key and we can use optional parameters for all our rest fancyness.